exporter/metric/metric.go

```diff
@@ -562,7 +560,7 @@ func lastValueToTypedValueAndTimestamp(lv *lastvalue.Aggregator, kind apimetric.
 	return tv, t, nil
 }
 
-func sumToTypedValueAndTimestamp(sum *sum.Aggregator, kind apimetric.NumberKind, start, end int64) (*monitoringpb.TypedValue, *monitoringpb.TimeInterval, error) {
+func sumToTypedValueAndTimestamp(sum *array.Aggregator, kind apimetric.NumberKind, start, end int64) (*monitoringpb.TypedValue, *monitoringpb.TimeInterval, error) {
```

Comment:
the function name should also change to `arrayToTypedValueAndTimestamp` probably?